qa/automation/ScoringPortal.Tests.Scenarios/1. Access to Portal/1.1. Access to Career Portal.feature

```diff
@@ -1,6 +1,67 @@
 Feature: 1.1. Access to Career Portal
 
-Scenario: 1.1_01 User can open Career Portal
-	Given 
-	When 
-	Then 
+Background:
+	Given user connected to the Internet using provided VPN profile
+
+Scenario: 1.1_01 User can open Quantori Scoring portal login page
+	Given User run <browser>
+	And used have not logged in the portal yet
+	When user enters URL https://qa.ddso-spot.quantori.com
+	Then user should see Quantori Scoring portal login page with following items
+		| Quantori Scoring logo      |
+		| Sign in with Google button |
+
+	Examples:
+		| browser |
+		| Chrome  |
+		| Edge    |
+		| Firefox |
+
+Scenario: 1.2_01 User can proceed to Google Authentication page
+	Given User opened Quantori Scoring portal login page
+	When user clicks on Sign in with Google
+	Then user should see Google authentication page
+
+Scenario: 1.2_02 User can login into Quantori Scoring portal using allowed Google account
+	Given User proceeded into Google Authentication page
+	When user enters valid email address from Qunatory domain

```

Comment:
You need to provide all necessary test data for the execution scenario. (specific email/ specific password etc)
This comment is valid for all similar mistakes.